src/app/about/page.tsx

```diff
@@ -1,23 +1,50 @@
 import { Container } from "@/components/shared/container";
 import { Metadata } from "next";
 
+interface SectionProps {
```

Comment:
also hot opinion but type suppremacy; if you dont have an insanely passionate view on interface can you change to `type Props = ...` lol

Usually named props only matter for stuff like importing types of functions but in this case i donut think we do that.

Reply:
Can you explain lol? Are interfaces bad???!